reports/reporting/__init__.py

```diff
@@ -47,3 +48,36 @@ def get_report_per_study(re_id, sd_id):
               f' for release {re_id} and study id {sd_id}')
 
     return jsonify(resp['Item']), 200
+
+
+@reports_api.route('studies/<sd_id>', methods=['GET'])
+def get_filter_by_state_study(sd_id):
+    endpoint_url = current_app.config['DYNAMO_ENDPOINT']
+    db = boto3.resource('dynamodb', endpoint_url=endpoint_url)
+    state = request.args.get('state')
+    study_table = db.Table(current_app.config['STUDY_SUMMARY_TABLE'])
+    # get the study summaries for the release and state
+    if state:
```

Comment:
It may be more concise to build a kwargs dict and optionally add the `FilterExpression` to it so you don't need to repeat the parameters

```
query = {
    "IndexName"'CreatedAtIndex',
    "Select:" 'ALL_PROJECTED_ATTRIBUTES',
    "KeyConditionExpression": Key('study_id').eq(sd_id),
    "ScanIndexForward":False
}

if state:
   query["FilterExpression"] = Attr('state').eq(state)
   
study_table.query(**query)
```